QuickTests/FunctionalTests/DefinitionsTests.swift

```diff
@@ -0,0 +1,54 @@
+import XCTest
+import Quick
+import Nimble
+
+private var fetches = [Int]()
+
+class FunctionalTests_DefinitionsSpec: QuickSpec {
+    override func spec() {
+        define("one") { return 1 }
+
+        it("should evaluate and return the defined variable") {
+            let one = fetch("one") as! Int
```

Comment:
Hmm... sorry to be negative, but I have doubts as to whether this addresses #231 in a satisfactory way. That issue criticizes how Quick recommends test setup that uses `!`. I doubt the author of that issue will appreciate the kind of forced casting you have here, which not only uses `!` but also performs a cast from `AnyObject`.

This approach works *around* the type system, instead of *leveraging* it, so it doesn't seem like "idiomatic" Swift.